### ISSUE TITLE
Global variables might not get parsed?

### ISSUE THREAD

**Issue by Manuelraa** [Manuelraa/yaml_folder_inventory#28]
Quick find from taking a closer look at the example.
Either spelling mistake or the variables are not getting parsed correctly
```
TASK [Print global haproxy group var] *******************************************************************************************************************************************************
ok: [internal-simu-haproxy-haproxy-db1] => {
    "haproxy_global_variable": "VARIABLE IS NOT DEFINED!"
}
ok: [internal-simu-haproxy-haproxy-db2] => {
    "haproxy_global_variable": "VARIABLE IS NOT DEFINED!"
}
ok: [external-prod-haproxy-haproxy-db1] => {
    "haproxy_global_variable": "VARIABLE IS NOT DEFINED!"
}
ok: [external-prod-haproxy-haproxy-db2] => {
    "haproxy_global_variable": "VARIABLE IS NOT DEFINED!"
}
```

**Comment by Manuelraa** [comment]
Yes indeed it is a  bug

**Comment by Manuelraa** [comment]
Fixed with release 1.2.2
